projects/batfish/test/unNestedDirectory/test3.cfg

```diff
@@ -0,0 +1 @@
+I am Test 3!!!
```

Comment:
Consider whether you can create the files in a temp directory inside of the test rather than check them in. I bet you can use the [`TemporaryFolder`](http://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html) to handle temp files including automatically deleting them when the test finishes.